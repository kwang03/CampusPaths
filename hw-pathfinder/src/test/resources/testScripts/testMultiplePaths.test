#Tests Dijkstra's when there are multiple paths with same jumps away but different weight costs

CreateGraph myGraph

AddNode myGraph A
AddNode myGraph B
AddNode myGraph C
AddNode myGraph D
AddNode myGraph E

AddEdge myGraph A B 2.0
AddEdge myGraph A C 1.0
AddEdge myGraph A D 1.0
AddEdge myGraph B E 2.0
AddEdge myGraph C E 1.0
AddEdge myGraph D E 2.0

FindPath myGraph A E