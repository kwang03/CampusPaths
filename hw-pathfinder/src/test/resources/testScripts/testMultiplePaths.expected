#Tests Dijkstra's when there are multiple paths with same jumps away but different weight costs

created graph myGraph

added node A to myGraph
added node B to myGraph
added node C to myGraph
added node D to myGraph
added node E to myGraph

added edge 2.000 from A to B in myGraph
added edge 1.000 from A to C in myGraph
added edge 1.000 from A to D in myGraph
added edge 2.000 from B to E in myGraph
added edge 1.000 from C to E in myGraph
added edge 2.000 from D to E in myGraph

path from A to E:
A to C with weight 1.000
C to E with weight 1.000
total cost: 2.000